app/assets/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -26,16 +26,25 @@ import PlaneView from "oxalis/view/plane_view";
 import constants, { OrthoViews, OrthoViewValues, OrthoViewValuesWithoutTDView } from "oxalis/constants";
 import type { Point2, Vector3, OrthoViewType, OrthoViewMapType } from "oxalis/constants";
 import type { ModifierKeys } from "libs/input";
-import { setViewportAction } from "oxalis/model/actions/view_mode_actions";
+import { setViewportAction, setTDCameraAction, zoomTDViewAction, moveTDViewXAction, moveTDViewYAction, moveTDViewByVectorAction } from "oxalis/model/actions/view_mode_actions";
+import { connect } from "react-redux";
+import api from "oxalis/api/internal_api";
+
+type OwnProps = {
+  onRender: () => void,
+};
+
+type Props = OwnProps & {
+  flycam: FlycamType,
+  scale: Vector3,
+};
 
 class PlaneController extends React.PureComponent {
   // See comment in Controller class on general controller architecture.
```

Comment:
Yes, I think so, we still have several controllers for the different tracing modes.